### ISSUE TITLE
Missing VoiceOver labels

### ISSUE THREAD

**Issue by isaackeyet** [wordpress-mobile/WordPress-iOS#651]
Got some feedback from a visually impaired user that pointed out some shortcomings in the app's function labels that help voiceover users navigate the app.

[ ] 'Navbar add" should be called something like "New Post". Take note for the posting button in the new tab bar.
[ ] HTML formatting links don't have labels, should have translatable labels ex "Bold Text", "Italic text", "Insert Link", "Quote Block".
